src/main/java/Sun.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Sun {
+    public static void main(String[] args) {
+        String logo = " ____               \n"
+                    + "| ___| _   _  ______ \n"
+                    + "| \\__ | | | || /--\\ |\n"
+                    + " \\___|| |_| || |  | |\n"
+                    + "/____/ \\__,_||_|  |_|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Sun!");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+        String command;
+        do{
```

Comment:
Maybe a space after 'do' would look nicer
```suggestion
        do {
```